src/discussions/topics/TopicsView.jsx

```diff
@@ -0,0 +1,54 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+import { useParams } from 'react-router';
+import { topicShape } from './topic-group/topic/Topic';
+import TopicGroup from './topic-group/TopicGroup';
+import TopicSearchBar from './topic-search-bar/TopicSearchBar';
+
+function TopicsView({ coursewareTopics, nonCoursewareTopics }) {
+  const { category } = useParams();
+
+  const renderNonCoursewareTopics = () => {
+    if (nonCoursewareTopics && category === undefined) {
+      return <TopicGroup subtopics={nonCoursewareTopics} />;
+    }
+
+    return (null);
+  };
+
+  const renderCoursewareTopics = () => {
+    const topics = (
+      category
+        ? coursewareTopics.filter(topic => topic.name === category)
+        : coursewareTopics
+    );
+
+    return topics.map(
+      topicGroup => (
+        <TopicGroup
+          id={topicGroup.id}
+          name={topicGroup.name}
+          subtopics={topicGroup.children}
+          key={topicGroup.name}
+        />
+      ),
+    );
+  };
+
+  return (
+    <div className="discussion-topics d-flex flex-column card">
```

Comment:
This is missing the filter beside the search bar (https://www.figma.com/proto/LVFnO9oRHYiamoicZuXDAC/Discussion-page?node-id=238%3A1676&scaling=min-zoom)